### ISSUE TITLE
Should optional field return null?

### ISSUE THREAD

**Issue by klarkc** [nwolverson/purescript-language-server#202]
I was attempting to trace the root cause of mattn/vim-lsp-settings#692, and I found that purescript-language-server is returning `NULL` for optional fields. Is this intended, and should the LSP client handle `NULL` as optional fields?

**Comment by nwolverson** [comment]
Yeah I imagine these should be absent rather than null, not sure if this is the lsp server itself or the library used. We should probably be using undefined instead of null in the representation, if that works out the same as absent by the time it gets to the wire
